Cargo.toml

```diff
@@ -49,7 +49,7 @@ serde_json = "1.0"
 serde = { version = "1.0", features = ["derive"] }
 sha2 = "0.10"
 thiserror = "1.0"
-tokio = { version = "1.21", features = ["macros", "fs"] }
+tokio = { version = "1.21", features = ["macros", "io-util"] }
```

Comment:
Just double checking, we don't need the fs feature? I thought we were using that somewhere in the code

Reply:
Yeah, just removing the feature doesn't seem to affect anything on my local machine (and I guess neither in CI). 

The io-util feature is needed for some async traits that fails compilation with the wasm32 target.